CHANGES.rst

```diff
@@ -9,6 +9,8 @@ Incompatible changes
 
 Deprecated
 ----------
+* #12630: Configuration defaults for ``linkcheck_allow_unauthorized`` and
+  ``linkcheck_report_timeouts_as_broken`` have been updated.
```

Comment:
Is this the right section? Or do you also intend to deprecate the config values themselves?

A

Reply:
I did wonder about that; it's (hopefully!) the end of the deprecation process for these, not the start of one -- but I did think it'd be worth mentioning in the changelog for upgraders.  Is there a better place to put them?  (none of the other categories initially seem suitable)

Reply:
Technically, *Incompatible changes*, as this is a backwards-incompatible change.

Reply:
Mmm; I'm on the fence but tempted to leave it in _Deprecated_ since it is deprecation-related.

Reply:
I view *Deprecated* as for new deprecations -- this is relating to an old deprecation. Note that e.g. changes in APIs go into *Incompatible changes* rather than *Deprecated*, when the deprecated behaviour gets removed.

A

Reply:
Ok.  I suppose theoretically the `timeout` status could cause code that's unaware of it to break.

Reply:
And I'm glad I re-read the entry as a result of that, because I've mis-stated the updated default.